2.0/db_models/pdf_sections.py

```
@@ -0,0 +1,20 @@
+from sqlalchemy import Column, Integer, String, Text, ForeignKey, JSON
+from sqlalchemy.orm import relationship
+from .base import Base
+import config.settings as settings
```

Pylint (3.3.7)

[refactor] Use 'from config import settings' instead

(R0402)